lib/travis/api/v3/models/cron.rb

```diff
@@ -0,0 +1,112 @@
+module Travis::API::V3
+  class Models::Cron < Model
+
+    belongs_to :branch
+
+    def start
+      raise ServerError, 'repository does not have a github_id'.freeze unless branch.repository.github_id
```

Comment:
Models have no knowledge that they are used as part of an HTTP request, should therefore not raise ServerError.